src/api/hooks/useGetDelegationNodeInfo.ts

```diff
@@ -21,27 +30,46 @@ export function useGetDelegationNodeInfo({
   validator,
 }: DelegationNodeInfoProps): DelegationNodeInfoResponse {
   const [state, _] = useGlobalState();
-  const [commission, setCommission] = useState<Types.MoveValue>();
-  const client = new AptosClient(state.network_value);
   const {totalVotingPower} = useGetValidatorSet();
+  const {data, isLoading} = useGetAccountResources(validatorAddress);
```

Comment:
not completely sure where `isLoading` coming from? doesn't seem that it is returned from `useGetAccountResources`
https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetAccountResources.ts#L19

Reply:
haha, `isLoading` is one of the returns from `useQuery`. a better example would be [here](https://github.com/aptos-labs/explorer/blob/b88dcecf3756b99afe9ea2ba37c6db7d4efcf970/src/api/hooks/useGetAccountResource.ts#L28) 


```
const accountResourcesResult = useQuery<
    Array<Types.MoveResource>,
    ResponseError
  >(
    ["accountResource", {address}, state.network_value],
    () => getAccountResources({address}, state.network_value),
    {refetchOnWindowFocus: false},
  );
  
  const {isLoading, isError, refetch} = accountResourcesResult;
```